tests/ssl-opt.sh

```diff
@@ -1120,6 +1125,11 @@ run_test() {
         return
     fi
 
+    if is_excluded_test_suite "${TEST_SUITE_NAME:-ssl-opt}"; then
+        SKIP_NEXT="NO"
```

Comment:
I'm pretty confused about why we appear to be specifically not skipping tests if they're excluded, but I suspect this may just be a quirk of the code.